utils/__init__.py

```diff
@@ -185,7 +189,10 @@ def deferred_verpick(version_d):
 
     @classproperty
     def getter(self):
-        return _version_pick(version_d)
+        if on_rtd:
```

Comment:
version picks return varying types, returning a string in the rtd case has breakage potential